CMake/resolve_dependency_modules/README.md

```diff
@@ -39,6 +39,7 @@ by Velox. See details on bundling below.
 | fbthrift          | v2024.04.01.00  | No       |
 | DuckDB (testing)  | 0.8.1           | Yes      |
 | cpr (testing)     | 1.10.15         | Yes      |
+| simdjson          | 3.8.0           | Yes      |
```

Comment:
I think simdjson already exists in the list above.

Reply:
That's right. Let me fix.